### PR TITLE
Update misspell to 0.5.1 and add `misspellings.csv`

### PR DESCRIPTION
**Pull request by silverwind**
Misspell 0.5.0 supports passing a csv file to extend the list of misspellings, so I added some common ones from the codebase. There is at least one typo in a API response so we need to decided whether to revert that and then likely remove the dict entry.

### CODE REVIEW

**Comment by silverwind** [comment]
The `committer` issue looks like a bug in misspell, filed https://github.com/golangci/misspell/issues/18.

**Comment by silverwind** [comment]
Should be ready now, the API change was reverted and the word "comiter" removed from the list of words. The only other potentially problematic change is the struct change for `Reviwer` property, but it seems it's not exposed externally.

File: assets/misspellings.csv [hunk 1]
```diff
@@ -0,0 +1,21 @@
+acounts,accounts
```
Comment (lunny) [inline assets/misspellings.csv]: Should we put the file under `build` directory?

Comment (silverwind) [reply assets/misspellings.csv]: It's basically a config file, could instead put it in root or [`.config`](https://github.com/pi0/config-dir), but the number of tools supporting the latter is slim.

Comment (silverwind) [reply assets/misspellings.csv]: I've moved it to `tools` now. At least it can't get confused with build assets now.

**Review by lunny — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**